sdk/python/feast/ui/package.json

```diff
@@ -6,7 +6,7 @@
     "@elastic/datemath": "^5.0.3",
     "@elastic/eui": "^57.0.0",
     "@emotion/react": "^11.9.0",
-    "@feast-dev/feast-ui": "^0.20.2",
+    "@feast-dev/feast-ui": "^0.20.3",
```

Comment:
Should this be bumped by the release process?

Reply:
it's kind of wonky right now and I have to fix it. NPM Publishes require an OTP so I'll have to fix the workflow